modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -149,6 +137,34 @@ the `..*` descendants selector on a key to select values from a flattened list.
 | Defines a function that expects to receive two inputs: a function to apply and an element to apply it on. The function is also recursively applied to the element's children. It uses `mapObject` `lower` `is` `if/else`.
 |===
 
+
+== DataWeave Examples in Mule
+
+These examples show how to use DataWeave to extract and transform data in Mule applications.
+
+[cols="2,5a",options="header"]
+|===
+|Example a| Description
+//change-script-output-mime
+|  xref:dataweave-cookbook-change-script-output-mime.adoc[Change a Script's Output Mime Type]
+| Shows how to customize the MIME type of a given format output, including an example of a JSON output with `application/problem+json` MIME type.
+//xlsx-lookup
+| xref:dataweave-cookbook-xlsx-lookup.adoc[Look Up Data in an Excel (XLSX) File]
+| Uses `filter` to return rows that contain a specified value.
+//csv-lookup
+| xref:dataweave-cookbook-csv-lookup.adoc[Look Up Data in CSV File]
+| Uses `filter` with `map` to look up data in a CSV file and return a country code for each calling code found within an input array of phone numbers.
```

Comment:
ditto: Shows how to use....
OR
perhaps the FIRST description should begin with "Customizes..." rather than "Shows how to customize"?
The goal is parallelism.